python/sglang/multimodal_gen/runtime/pipelines_core/stages/denoising.py

```diff
@@ -151,6 +143,26 @@ def __init__(
         # misc
         self.profiler = None
 
+    def torch_compile_module(self, module):
+        """
+        Compile a module's forward with torch.compile, and enable inductor overlap tweak if available.
+        No-op if torch compile is disabled or the object has no forward.
+        """
+        if not self.server_args.enable_torch_compile or module is None:
+            return module
+        if not hasattr(module, "forward"):
+            return module
+        try:
+            import torch._inductor.config as _inductor_cfg
+
+            _inductor_cfg.reorder_for_compute_comm_overlap = True
+        except ImportError:
+            pass
+        mode = os.environ.get("SGLANG_TORCH_COMPILE_MODE", "max-autotune-no-cudagraphs")
+        compiled_forward = torch.compile(getattr(module, "forward"), mode=mode)
```

Comment:
Add a log before this line is better? Such as `logger.info(f"Compiling transformer with mode: {mode}")`